src/test/java/com/apifest/doclet/tests/resources/ParsingResource.java

```diff
@@ -0,0 +1,64 @@
+package com.apifest.doclet.tests.resources;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
```

Comment:
As this class is used as a resource to the doclet tests (the code does not need to be compiled), it's better to place it under src/test/resources. Also, the name of the class implies that the class will parse a resource not that it will be parsed. May be "TestResource" or something like that will be more appropriate.